src/browser/stores/dota.js

```diff
@@ -109,6 +112,38 @@ const DotaStore = Fluxxor.createStore({
         const leagueIds = this.patchIdsToLeagueIds[patchId];
 
         return leagueIds.map(id => this.leagueIdsToLeagueNames[id]);
+    },
+
+    getWinrate(combination) {   
+        let winCounter = 0;
+        let matchCounter = 0;
+
+        const matches = this.getMatches(combination.matches);
+
+        //just need id of one hero, if one hero is there then whole combo too
+        const targetHeroId = combination.heroIds[0];
+        
+        //tally up the wins
+        matches.forEach(match => {
+            matchCounter++;
+
+            //check if combo appears on radi or dire, assume radi
+            let radiOrDire = 1;
```

Comment:
combinationSideId?   I'm trying to think of a name that communicates the variable a little more